respa_admin/static_src/js/resourceFormPeriods.js

```diff
@@ -78,7 +108,16 @@ function updatePeriodDaysIndices($periodItem) {
         $(cellInput).attr('name', $(cellInput).attr('name').replace(/-(\d+)-(\d+)-/, '-' + periodIdNum + '-' + dayIndex + '-'));
       });
     })
-  });
+  };
+
+  let index = 0;
```

Comment:
Maybe a comment on why the `index` variable is defined outside the for-loop would be good for someone reading the code in the future, it confused me at first.

Reply:
This was indeed unlegible code. It's clearer now.